plotter/boxplot.go

```diff
@@ -64,6 +65,10 @@ type BoxPlot struct {
 	// GlyphStyle is the style of the outside point glyphs.
 	GlyphStyle draw.GlyphStyle
 
+	// FillColor is the color used to fill the box.
+	// Default value is nil
```

Comment:
Sorry, I missed this. The default is nil by consequence of the language, so having a semantic doc would be better `// The default is no fill.` (also note doc sentences always have trailing periods).

Reply:
Agreed. Updated